### PR TITLE
plugin.audio.calmradio v0.2.6

### PR DESCRIPTION
**Pull request by nire0510**
### Description
- Fix favorites API endpoint
- Add previous and next songs### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a sinlge commit with using the following style: [plugin.video.foo] v1.0.0

Addtional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuesly monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: plugin.audio.calmradio/artwork.py [hunk 1]
```diff
@@ -29,13 +32,17 @@ def __init__(self):
         self.channel = xbmcgui.ControlLabel(810, 160, 1060, 30, 'Channel Name', textColor='0xff18b0e2')
 
         # channel description
-        self.description = xbmcgui.ControlTextBox(810, 200, 1060, 800, 'Channel Description')
+        self.description = xbmcgui.ControlTextBox(810, 200, 1060, 350, 'Channel Description')
```
Comment (phil65) [inline plugin.audio.calmradio/artwork.py]: please also translate this one for next release.

**Comment by phil65** [comment]
please take care of the comments for next release, ok? :)

File: plugin.audio.calmradio/changelog.txt [hunk 2]
```diff
@@ -1,7 +1,11 @@
+v0.2.6
+- Fix favorites API endpoint
+- Add previous and next songs
+
 v0.2.5
 - fix missing bg image bug
 
-v0.2.4
+-v0.2.4
```
Comment (phil65) [inline plugin.audio.calmradio/changelog.txt]: this one here seems also wrong. ;)

**Comment by nire0510** [comment]
Hi @phil65 , no problem - I took care for all of your comments above, they'll be part of next version. BTW, we were working together almost a week ago on [another PR I've created](https://github.com/xbmc/repo-scripts/pull/281). You were about to merge it but then asked me to move it to `repo-scripts`. Can you take a look and see if it can be also merged?
Thanks!

**Pull request merged**